### PR TITLE
Update the `dist pull` Command Description

### PR DESCRIPTION
**Pull request by praneesha**
## Purpose
Update the `dist pull` command description in the "Pull a specific distribution" section[1].

[1] https://ballerina.io/learn/keeping-ballerina-up-to-date/#pull-a-specific-distribution

> Fixes https://github.com/ballerina-platform/ballerina-dev-website/issues/1108

## Check List

- [ ] **Page Addition**
  - [ ] Add `permalink` to pages
  - [ ] If contains empty folder(s), Add front-matter `redirect_to:`

- [ ] **Page Rename**
  - [ ] Add front-matter `redirect_from`
  - [ ] Add front-matter `redirect_to:` (If applicable)

### CODE REVIEW

File: swan-lake/learn/keeping-ballerina-up-to-date.md [hunk 1]
```diff
@@ -233,12 +233,22 @@ Distributions available locally:
 
 ### Pull a specific distribution
 
-- The `ballerina dist pull <distribution>` command downloads a particular distribution and stores it in your local environment. It also sets the fetched distribution as the active distribution.
+The `ballerina dist pull <distribution>` command downloads a particular distribution and stores it in your local environment. It also sets the fetched distribution as the active distribution.
+
+#### For versions below jBallerina 1.2.5 (for update tool versions below 0.8.8):
```
Comment (keizer619) [inline swan-lake/learn/keeping-ballerina-up-to-date.md:238]: @praneesha  shall we interchange the order as 0.8.8 will be packed by default?

File: swan-lake/learn/keeping-ballerina-up-to-date.md [hunk 2]
```diff
@@ -233,12 +233,22 @@ Distributions available locally:
...
+
+```sh
+→ sudo ballerina dist pull jballerina-1.0.3
```
Comment (keizer619) [inline swan-lake/learn/keeping-ballerina-up-to-date.md:241]: Lets include pulling a new version

File: swan-lake/learn/keeping-ballerina-up-to-date.md [hunk 3]
```diff
@@ -233,12 +233,22 @@ Distributions available locally:
...
+```
+
+#### For jBallerina 1.2.5 and above (for update tool version 0.8.8 and above):
 
 ```sh
-→ sudo ballerina dist pull 1.0.3
-Fetching the '1.0.3' distribution from the remote server...
-Downloading 1.0.3 100% [==================================] 96/96 MB '1.0.3' successfully set as the active distribution
+→ sudo ballerina dist pull 1.2.6
+Fetching the 'jballerina-1.0.3' distribution from the remote server...
```
Comment (keizer619) [inline swan-lake/learn/keeping-ballerina-up-to-date.md:250]: 1.2.6

Comment (praneesha) [reply swan-lake/learn/keeping-ballerina-up-to-date.md:250]: Fixed.

File: swan-lake/learn/keeping-ballerina-up-to-date.md (hunk 1 above)
Comment (praneesha) [reply swan-lake/learn/keeping-ballerina-up-to-date.md:238]: Done.

File: swan-lake/learn/keeping-ballerina-up-to-date.md (hunk 2 above)
Comment (praneesha) [reply swan-lake/learn/keeping-ballerina-up-to-date.md]: Changed to 1.2.4.

**Pull request merged**